setup.cfg

```diff
@@ -68,8 +68,11 @@ force_grid_wrap=0
 combine_as_imports=True
 
 [mypy]
-ignore_missing_imports = True
 plugins = pydantic.mypy
+follow_imports = silent
+ignore_missing_imports = True
+allow_redefinition = True
+#check_untyped_defs = True
```

Comment:
I also want mypy to be stricter but I think it should not be done in a fix. While you are it, can be try with `strict = True` ?

Reply:
If mypy enabled on `peakina` only:
- 0 errors with check_untyped_defs
- 106 errors with strict
If mypy enabled on `.`:
- 26 errors with check_untyped_defs
- 229 errors with strict